ynr/apps/moderation_queue/tests/test_changes_for_review.py

```diff
@@ -257,6 +258,85 @@ def test_users_with_past_approved_edits_dont_need_review(self):
             PREVIOUSLY_APPROVED_COUNT + 1,
         )
 
+    def update_person_and_version(self, user, example_person):
+        response = self.app.get(
+            "/person/{person_id}/update".format(person_id=example_person.id),
+            user=self.user,
+        )
+
+        form_data = response.forms["person-details"]
+        form_data["favourite_biscuit"] = "Jammy Dodgers"
+        form_data.submit()
+
+        LoggedAction.objects.create(
+            user=self.user,
+            action_type=ActionType.PERSON_UPDATE,
+            person=example_person,
+            popit_person_new_version=random_person_id(),
+            source="Just for tests...",
+        )
+        example_person.refresh_from_db()
+        example_person.favourite_biscuit = "Jammy Dodgers"
+        example_person.record_version(get_change_metadata(None, "Biscuit edit"))
```

Comment:
Similarly to the above, you shouldn't need to do this explicitly as it is handled by the UpdatePersonView https://github.com/DemocracyClub/yournextrepresentative/blob/7a1a61ed7db11965c5fc6ab6973fe749c874b300/ynr/apps/candidates/views/people.py#L472